.github/workflows/static.yml

```diff
@@ -0,0 +1,44 @@
+# Simple workflow for deploying static content to GitHub Pages
```

Comment:
I like the github actions integration very much. However, we are using azure pipelines and a different provider for the site hosting (will deploy it to an azure CDN point).
You can leave this, this is more for me to disable this action later.

Reply:
Yeah, I should have used another branch for the PR.